pynws/const.py

```diff
@@ -60,3 +61,65 @@
     "blizzard": "Blizzard",
     "fog": "Fog/mist",
 }
+
+
+class Detail(StrEnum):
```

Comment:
I like how this reads when referenced (e.g., Detail.TEMPERATURE). Maybe we can do the same for the SimpleForecast properties (e.g., Simple.TEMPERATURE). Thoughts?

Reply:
The only comment I have is whether we should structure/name this to indicate that it is for forecasts.  What happens when/if we add an StrEnum for the observations?

We could have `Detail`, `Simple`, `Observation`?

Reply:
Either way is fine with me, but I have a slight preference for the terser versions (Detail and Simple).